### DISCUSSION TITLE
How can I get a total result after running the tests?

### DISCUSSION THREAD

**Discussion by TifYang2** [google/googletest#4629] [Community Help]
After I running the test, I get a gtest.log maybe, but how can I get a message in the gtest.log like "All tests passed" or I need to check the log to see whether it works.

**Comment by AKA-MMM** [comment]
Hi,
a simple way is to generate a report (for instance in XML format). For this, you can use the following main:
int main(int argc, char **argv) {
    testing::GTEST_FLAG(output) = "xml:./TestReport.xml";
    testing::InitGoogleTest(&argc, argv);
    return RUN_ALL_TESTS();
}
